### DISCUSSION TITLE
Is it possible to run models for just one city in a country?

### DISCUSSION THREAD

**Discussion by dping590** [pypsa-meets-earth/pypsa-earth#1070] [General]
Sorry I'm a just a beginner? I'm wondering is it possible to run a model with only one COUNTY's boroughs in a large country?

**Comment by pz-max** [comment]
@dping590 Yes, but this currently requires rather expert knowledge as it is not supported by default.
We would need to build scalable solutions to address that for the whole OS community.

**Reply by dping590** [reply]
Thanks a lot for the answer!

**Comment by ekatef** [comment]
Hello @dping590!

Adding to what @pz-max explained, there is [PyPSA-Distribution](https://github.com/pypsa-meets-earth/pypsa-distribution) package which goal is to simulate small-scale applications building on top of PyPSA-Earth. Distribution grids are perfectly in scope, and the urban scale can absolutely be addressed.

If you are interested feel free to join PyPSA-Distibution biweekly meetings which are open for everyone interested. The calendar invitation can be found under [Get involved](https://github.com/pypsa-meets-earth/pypsa-distribution?tab=readme-ov-file#get-involved) section of README.

**Reply by dping590** [reply]
Thanks a lot for the answer!
